src/components/AppLayout/Footer.tsx

```diff
@@ -46,84 +29,163 @@ const Footer = ({ className } : { className?:string }) => {
 									<div>
 										<a href='https://polkassembly.hellonext.co/' target='_blank' rel='noreferrer'>
 											Report an Issue
+											<RedirectIcon className='ml-3' />
 										</a>
 									</div>
-									<div>
+									<div className='max-[650px]:mb-[-5px]'>
 										<a href='https://feedback.polkassembly.io' target='_blank' rel='noreferrer'>
 											Feedback
+											<RedirectIcon className='ml-3' />
 										</a>
 									</div>
+									<div className='max-[650px]:hidden'>
+										<Link href='/terms-and-conditions'>
+											Terms and Conditions
+										</Link>
+									</div>
 									<div>
 										<a href='https://github.com/polkassembly/polkassembly' target='_blank' rel='noreferrer'>
 											Github
+											<RedirectIcon className='ml-3' />
 										</a>
 									</div>
 								</div>
 							</nav>
 						</div>
 
-						<div className="mt-10 md:mt-0 md:ml-10 lg:ml-14 text-center sm:text-left">
-							<p className="text-lg font-medium text-sidebarBlue">Our Services</p>
+						<div className="mt-8 md:mt-0 md:ml-10 lg:ml-14 text-center sm:text-left text-[#485F7D] font-normal">
+							<p className="text-lg font-bold text-[#243A57] mb-[-5px] max-[650px]:mt-[-15px] ">Our Services</p>
 
-							<nav aria-label="Footer Services Nav" className="mt-4 md:mt-8">
-								<div className="space-y-4 text-sm">
+							<nav aria-label="Footer Services Nav" className="mt-4 md:mt-3">
+								<div className="space-y-2 text-sm">
 									<div>
+										<a href='https://docs.polkassembly.io/' target='_blank' rel='noreferrer'>
+										Docs
+										</a>
+										<RedirectIcon className='ml-3' />
+									</div>
+
+									<div className='max-[650px]:hidden'>
 										<Link href={'/terms-of-website'} >
 											Terms of Website
 										</Link>
 									</div>
 
-									<div>
+									<div className='max-[650px]:hidden'>
 										<Link href={'/privacy'}>
 											Privacy Policy
 										</Link>
 									</div>
-									<div>
-										<a href='https://docs.polkassembly.io/' target='_blank' rel='noreferrer'>
-										Docs
-										</a>
-									</div>
-
 								</div>
 							</nav>
 						</div>
 					</div>
 				</div>
 
 				{/* Below divider */}
-				<div className="mt-12 border-t border-gray-100 pt-6">
+				<Divider className='mb-0' />
+				<div className="mt-5 pb-3 text-sm text-[#485F7D] font-medium">
 					<div className="text-center sm:flex sm:justify-between sm:text-left">
-						<p className="text-sm text-gray-500">
+						<div className='flex max-[650px]:flex-col'>
+							<p className=" max-[650px]:mb-0 mr-1 ">
+							A House of Commons Initiative.
+							</p>
+							<p className="mt-4  sm:mt-0 ">
+						Polka Labs Private Limited {new Date().getFullYear()}
+							</p>
+						</div>
+
+						<p className="mb-0">
 							<span className="block sm:inline">All rights reserved.</span>
 						</p>
+					</div>
+				</div>
+				<div className='md:hidden'>
```

Comment:
we don't need whole another  div for small devices, can be done in the div above